src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+“Oh, and last night I had a sexy dream about the mom from Family Guy. Don’t tell nobody.”
+
+[[I made a mental note to never ask Darryl a personal question again|The Gutter - Bar - Darryl]].
+
+::The Gutter - Bar - Darryl toe
+“Well...” Darryl paused for dramatic effect, or maybe he forgot the story.
+
+“They call him Toe because he has none. He cut them off and tried to eat them during a nasty trip, the story goes.
+
+“I’ve heard he used to love to have chicks suck his toes and now he pays to have them suck the air where they used to be.”
+
+Toe loomed over us both and we shrank. He nodded gravely and cleared Darryl’s empty glass.
+
+As he was turning away he let out a low rumble. “It’s all true.”
+
+[[I had solved another mystery. Good practice, I guess|The Gutter - Bar - Darryl]]?
+
+::The Gutter - Bar - Darryl ron
+“That’s between me, Ron, and the packet of hot sauce he tried to shove down my trousers as a prank.” Darryl said plainly.
+
+“Oh, dangit.” Darryl shook his head. “I goofed.”
+
+[[I laughed very loud at Darryl’s goof|The Gutter - Bar - Darryl]].
+
+::The Gutter - Ron
+Sitting down next to Ron at the bar seemed weird. I just stood behind him instead, stuck for a moment on what to say.
+
+>>[["Hey. Fuck you, Ron."|The Gutter - Ron - Fuck you]]
+>>[["Uh, Ron, can I talk to you out back?"|The Gutter - Ron - Request to talk]]
+>>[[I pretended to shake Ron’s hand, then pulled back at the last second to fix my hair. “Too slow, fucko,” I said as I walked away smiling. It never gets old|The Gutter - Inside hub]].
+
+::The Gutter - Ron - Request to talk
+Ron was tipping back the last of his beer as I spoke.
+
+"Hahhhhhhh…. Hell naw! Unless this fuck got a tap for me out there," he said, gesturing at the bartender with his beer mug.
+
+I sighed. [["Oh fuck you, Ron."|The Gutter - Ron - Fuck you]]
+I tossed up my hands. [[“Fuck this. Not worth it.”|The Gutter - Inside hub]]
+
+::The Gutter - Ron - Fuck you
+"Ah, fuck me, eh? Fuck your old man Ron?" He leered into the foamy residue of his drained beer. "Nah. Your old lady takes care of that plenty. Whaddaya need, runt?"
+
+"Ron. You gotta tell me where Anna went."
+
+Ron turned on his stool and squinted at me out of the corner of his eye.
+
+"Hm. Might be. Might be." He eyed me levelly. "Might be that this—" he patted a rough-looking envelope sticking out of his shirt pocket "—is something I could pass on to ya." He leaned toward me. I caught the reek of spilt beer and stale cigarettes on his clothes.
+
+(link: "...")[$dissolve["Well?" I said.
+
+(display: "The Gutter - Ron - Pool request")]]
+
+(set: $talkedToRon to true)
+
+::The Gutter - Ron - Pool request
+"Well, what the hell you got for your fucking shithead papa Ron, hm?" He looked over at the pool table as he spoke, like the table had an itch he needed to scratch.
+
+“Wanna play a game?” I asked through clenched teeth.
+[I shrugged. [[Whatever|The Gutter - Inside hub]].]<whatever|
```

Comment:
This option should disappear if you click "Wanna play a game?" because otherwise it looks kinda weird:

![screen shot 2018-08-26 at 9 36 52 am](https://user-images.githubusercontent.com/4277237/44630499-98ede380-a913-11e8-874e-8e5bf6e4f62f.png)

Reply:
Might be easiest to just bump the game stuff into its own passage.

Reply:
Fixed! I was missing the replace command for once you've cleared the pool table.